src/components/Modal/Plugins.vue

```diff
@@ -43,7 +43,7 @@
         </UiButton>
       </div>
     </template>
-    <div v-else class="m-4 p-4 border rounded-2 text-white">
+    <div v-if="selected !== false" class="m-4 p-4 border rounded-2 text-white">
```

Comment:
Why not using v-else anymore here?

Reply:
Somehow Vue 3 doesn't register the prepending `v-for` in the `div`, so the v-else wont work. Maybe it has something to do with that it's in a template tag.

Reply:
I thought that was the simplest solution.